database/migrations/2023_11_22_121149_create_values_table.php

```diff
@@ -0,0 +1,164 @@
+            $table->char('object_id', 36)->collation('ascii_bin');
+
+            $table->char('position_id', 36)->collation('ascii_bin')->nullable();
+            $table->char('ref_object_id', 36)->collation('ascii_bin')->nullable();
+            $table->string('string_value')->nullable();
+            $table->integer('int_value')->nullable();
+            $table->dateTime('datetime_value')->nullable();
+
+            $table->char('created_by', 36)->collation('ascii_bin');
+            $table->dateTime('created_at');
+            $table->dateTime('updated_at');
+
+            $table->foreign('attribute_id')->references('id')->on('attributes');
+            $table->index('object_id');
+            $table->foreign('position_id')->references('id')->on('positions');
+            $table->index('ref_object_id');
+            $table->foreign('created_by')->references('id')->on('users');
+        });
+
+        $date = '2023-11-22 00:00:00';
+        $systemUserId = 'e144ff18-471f-456f-a1c2-971d88b3d213';
+        DB::table('attributes')->upsert([
+            [
+                'id' => '540e2b26-1330-42ae-8209-55cf22bb3638',
+                'facility_id' => null,
+                'table' => 'positions',
+                'name' => 'duration_minutes',
+                'api_name' => 'duration_minutes',
+                'type' => 'int',
+                'dictionary_id' => null,
+                'default_order' => 2,
+                'is_multi_value' => 0,
+                'requirement_level' => 'empty',
+                'created_at' => $date,
+                'updated_at' => $date,
+            ],
+            [
+                'id' => 'c9ab3795-0012-4cfe-8100-a7bb1dd9755b',
+                'facility_id' => null,
+                'table' => 'positions',
+                'name' => 'category',
+                'api_name' => 'category_dict_id',
+                'type' => 'dict',
+                'dictionary_id' => 'ce12aa23-a5db-49f3-987b-d2db3ab24a3b',
+                'default_order' => 1,
+                'is_multi_value' => 0,
+                'requirement_level' => 'empty',
+                'created_at' => $date,
+                'updated_at' => $date,
+            ],
+            [
+                'id' => 'c97a3c3b-7faa-4f51-9113-2390ebaba700',
+                'facility_id' => null,
+                'table' => 'dictionaries',
+                'name' => 'position_required_attributes',
+                'api_name' => 'position_required_attributes',
+                'type' => 'attributes',
+                'dictionary_id' => null,
+                'default_order' => 1,
+                'is_multi_value' => 1,
+                'requirement_level' => 'optional',
+                'created_at' => $date,
+                'updated_at' => $date,
+            ],
+        ], 'id');
+
+        DB::table('values')->upsert([
+            [
+                'id' => '2a58e858-721b-4e8c-8d18-f419c408fc42',
```

Comment:
Tu by chyba nie zawadził komentarz co to jest co tu wstawiasz bo naprawdę dość trudno zgadnąć. No chyba że jest jakaś konwencja żeby pliki migracji były możliwie kryptyczne :)

Reply:
no dane wstawiam, nie wiem co mam  napisać więcej

Reply:
historia była taka:
- wstawiłem te dane do tabeli ręcznie, tak, żeby mi to działało
- wyeksportowałem dane z tabeli do tablicy php
- wstawiłem tę tablicę do migracji
nie ma tu szczególnie co się rozpisywać co się wydarzyło